app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunHeader/RunHeaderContent/ActionButton/index.tsx

```diff
@@ -133,6 +133,7 @@ export function ActionButton(props: ActionButtonProps): JSX.Element {
         }
         onClick={handleButtonClick}
         id="ProtocolRunHeader_runControlButton"
+        borderRadius="200px"
```

Comment:
```suggestion
        borderRadius={BORDERS.borderRadiusFull}
```
We do need to update the `StyledText` here in the same manner as the Cancel button. The font is a bit different now.